components/CountryDetails/index.js

```diff
@@ -0,0 +1,67 @@
+import StyledFlag from "../StyledFlag";
+import StyledDiv from "../StyledDiv";
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledLink = styled.a`
+  font-size: 2rem;
+  position: relative;
+  right: 9rem;
+  top: 1rem;
+`;
+
+export default function CountryDetails({ selectedCountry }) {
+  function addSeparators(number) {
+    return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ".");
+  }
+  if (!selectedCountry) return <h2>is Loading</h2>;
+  const languagesWithSeparators =
+    selectedCountry.languages &&
+    Object.values(selectedCountry.languages).join(", ");
+
+  const timezoneWithSeparators = selectedCountry.timezones?.join(", ");
+
+  return (
+    <>
+      <StyledDiv isOnDetailsPage>
+        <StyledLink href="/list">🔙</StyledLink>
+        <h1>{selectedCountry.name.common}</h1>
+        <p>
+          <strong>Continent:</strong> {selectedCountry.continents}
+        </p>
+        <p>
+          <strong>Subregion:</strong> {selectedCountry.subregion}
+        </p>
+        <p>
+          <strong>Capital:</strong> {selectedCountry.capital}
+        </p>
+        <p>
+          <strong>Population:</strong>{" "}
+          {addSeparators(selectedCountry.population)}
+        </p>
+        <p>
+          <strong>Language: </strong>
+          {languagesWithSeparators}
+        </p>
+
+        {
```

Comment:
Not sure but I think you can remove the `{...}` which are wrapping the `p` since the ternary operator is only affecting how the currency is shown after `Currency:`

Reply:
No, thats not working, when I remove the {...} then the page shows the text languagesWithSeperators i.o. the actual languages

Reply:
Oh, my bad in this case and sorry for the circumstances!